hw/bsp/dialog_da1469x-dk-pro/src/hal_bsp.c

```diff
@@ -41,6 +41,15 @@ static const struct hal_bsp_mem_dump dump_cfg[] = {
  */
 static char hw_id[] = "DA1469X_HW_ID";
 
+
+/*
+ * Configure empty RDIDs for use by flash init.
+ */
+#if MYNEWT_VAL(RAM_RESIDENT)
+const struct qspi_flash_config rdids[] = 0;
+const int qspi_flash_config_array_size = ARRAY_SIZE(rdids);
```

Comment:
```
Error: repos/apache-mynewt-core/hw/bsp/dialog_da1469x-dk-pro/src/hal_bsp.c:49:42: error: invalid initializer
 const struct qspi_flash_config rdids[] = 0;
                                          ^
repos/apache-mynewt-core/hw/bsp/dialog_da1469x-dk-pro/src/hal_bsp.c:50:42: error: implicit declaration of function 'ARRAY_SIZE'; did you mean 'RAM_SIZE'? [-Werror=implicit-function-declaration]
 const int qspi_flash_config_array_size = ARRAY_SIZE(rdids);
                                          ^~~~~~~~~~
                                          RAM_SIZE
repos/apache-mynewt-core/hw/bsp/dialog_da1469x-dk-pro/src/hal_bsp.c:50:42: error: initializer element is not constant
```
I guess you meant:
```c
const struct qspi_flash_config rdids[] = {};
```

Reply:
Either works, but I'll update that to {}.